tools/regionalGAM/scriptsR_Galaxy/flight_curve.R

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env Rscript
 #library('getopt')
 #library(devtools)
+library(data.table)
 
 args = commandArgs(trailingOnly=TRUE)
 source(args[1]) #TODO replace by library(regionalGAM) if available as official package from bioconda
 
-tryCatch({input = read.table(args[2], header=TRUE,sep="	")},finally={input = read.table(args[2], header=TRUE,sep=",")})
-dataset1 <- input[,c("SPECIES", "SITE", "YEAR", "MONTH", "DAY", "COUNT")]
+tryCatch({input = read.table(args[2], header = TRUE,sep=" ")},
```

Comment:
Just `input = fread(args[2], header = TRUE)` should do it
Without `trycatch` and `read.table`